mobile/lib/modules/asset_viewer/ui/exif_bottom_sheet.dart

```diff
@@ -102,84 +93,83 @@ class ExifBottomSheet extends ConsumerWidget {
           ),
 
           // Location
-          assetDetail.exifInfo?.latitude != null
-              ? Padding(
-                  padding: const EdgeInsets.only(top: 32.0),
-                  child: Column(
-                    crossAxisAlignment: CrossAxisAlignment.start,
-                    children: [
-                      Divider(
-                        thickness: 1,
-                        color: Colors.grey[600],
-                      ),
-                      Text(
-                        "LOCATION",
-                        style: TextStyle(fontSize: 11, color: Colors.grey[400]),
-                      ),
-                      _buildMap(),
-                      _buildLocationText(),
-                      Text(
-                        "${assetDetail.exifInfo!.latitude!.toStringAsFixed(4)}, ${assetDetail.exifInfo!.longitude!.toStringAsFixed(4)}",
-                        style: TextStyle(fontSize: 12, color: Colors.grey[400]),
-                      )
-                    ],
+          if (assetDetail.exifInfo?.latitude != null)
+            Padding(
+              padding: const EdgeInsets.only(top: 32.0),
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  Divider(
+                    thickness: 1,
+                    color: Colors.grey[600],
                   ),
-                )
-              : Container(),
+                  Text(
+                    "LOCATION",
+                    style: TextStyle(fontSize: 11, color: Colors.grey[400]),
+                  ),
+                  if (assetDetail.exifInfo!.latitude != null &&

```

Comment:
I like this, the function `_build*` will just act as building things without worry about additional condition.